src/libs/ajax/Catalog.ts

```diff
@@ -121,7 +121,7 @@ export interface ExportDatasetRequest {
 }
 
 export interface CatalogContract {
-  getDatasets: () => Promise<DatasetTableResponse>
+  getDatasets: () => Promise<DatasetListResponse>
```

Comment:
Interesting, I guess we got the types wrong here!

Reply:
Yeah, now that the types are being exercised I caught a few bugs.